torch_np/_normalizations.py

```diff
@@ -152,7 +156,31 @@ def wrapped(*args, **kwds):
                 )
             # finally, pass normalized arguments through
             result = func(*ba.args, **ba.kwargs)
-            return result
+
+            # handle returns
+            r = sig.return_annotation
+            if r == NDArray:
+                return _helpers.array_from(result)
+            elif r == inspect._empty:
+                return result
+            elif hasattr(r, "__origin__") and r.__origin__ in (list, tuple):
+                # this is tuple[NDArray] or list[NDArray]
+                # XXX: change to separate tuple and list normalizers?
+                return r.__origin__(_helpers.tuple_arrays_from(result))
```

Comment:
I'm actually not sure if this is the right thing to do. The goal here is to get a `tuple` from `tuple[NDArray]`.